_includes/templates/primary-nav.html

```diff
@@ -1,4 +1,5 @@
 {% import "nav-items-toplevel.html" as vars with context %}
+<div class="sliding-nav_nav">
```

Comment:
Yeah this is worth looking into. IIRC there's even an additional wrapper around when this template is included.